### PR TITLE
feat: Add adminer

### PR DESCRIPTION
**Pull request by bharath-123**
[Adminer](https://www.adminer.org/) is a very useful tool for visually inspecting the MEV relayer's database entries.

### CODE REVIEW

**Review by Makpara — APPROVED** [review]

File: src/mev_relay/mev_relay_launcher.star [hunk 1]
```diff
@@ -1,6 +1,10 @@
 redis_module = import_module("github.com/kurtosis-tech/redis-package/main.star")
+<<<<<<< HEAD
```
Comment (h4ck3rk3y) [inline src/mev_relay/mev_relay_launcher.star:2]: there's a merge conflict here!!

**Review by h4ck3rk3y — CHANGES_REQUESTED** [review]

File: src/mev_relay/mev_relay_launcher.star (hunk 1 above)
Comment (bharath-123) [reply src/mev_relay/mev_relay_launcher.star]: fixed!

File: src/mev_relay/mev_relay_launcher.star [hunk 2]
```diff
@@ -1,5 +1,5 @@
 redis_module = import_module("github.com/kurtosis-tech/redis-package/main.star")
-postgres_module = import_module("github.com/kurtosis-tech/postgres-package/main.star")
+postgres_module = import_module("github.com/bharath-123/postgres-package/main.star@add-adminer")
```
Comment (h4ck3rk3y) [inline src/mev_relay/mev_relay_launcher.star:2]: can you fix this?

**Review by h4ck3rk3y — CHANGES_REQUESTED** [review]

**Comment by bharath-123** [comment]
hey this is fixed!

**Review by h4ck3rk3y — APPROVED** [review]

File: src/mev_relay/mev_relay_launcher.star [hunk 3]
```diff
@@ -38,7 +38,7 @@ def launch_mev_relay(
         user="postgres",
         database="postgres",
         service_name="postgres",
-        persistent=DONT_PERSIST_TO_DISK,
```
Comment (h4ck3rk3y) [inline src/mev_relay/mev_relay_launcher.star:41]: why are you removing this?

File: src/mev_relay/mev_relay_launcher.star [hunk 4]
```diff
@@ -38,7 +38,7 @@ def launch_mev_relay(
...
+        launch_adminer=True,
```
Comment (h4ck3rk3y) [inline src/mev_relay/mev_relay_launcher.star:41]: can you take the `True` out as a constant?

**Review by h4ck3rk3y — CHANGES_REQUESTED** [review]

**Review by h4ck3rk3y — APPROVED** [review]

**Pull request merged**